<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loginTab">
      <attribute name="title">
       <string>Форма входа</string>
      </attribute>
      <layout class="QVBoxLayout" name="loginTabLayout">
       <item>
        <widget class="QWidget" name="loginContainerWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="logoLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Картинка</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="errorMsgLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Сообщение об ошибке</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="loginIcoLabel">
              <property name="text">
               <string>ico</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="loginEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="passwordIcoLabel">
              <property name="text">
               <string>Ico</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="passwordEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="passwordModeButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QPushButton" name="canselButton">
              <property name="text">
               <string>Отмена</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="loginButton">
              <property name="text">
               <string>Войти</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QPushButton" name="repasswordButton">
              <property name="text">
               <string>Я забыл пароль</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="newUserButton">
              <property name="text">
               <string>Новый пользователь</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="calendarTab">
      <attribute name="title">
       <string>Календарь</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="itemTab">
      <attribute name="title">
       <string>Тренировка</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Забыл пароль</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Новый пользователь</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="msgFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>loginEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>canselButton</tabstop>
  <tabstop>passwordModeButton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
